templates/projects/tasks_browse.webpack.ejs

```diff
@@ -247,9 +247,23 @@
                 {% endif %}
                 {% if 'gold_task' in filter_data.display_columns %}
                     {% if t.calibration %}
-                    <td style="width: 100%"><a class="label label-warning" style="display: block; margin: 0 0.25rem" target="_blank" href="{{ url_for('project.task_presenter', short_name=project.short_name, task_id=t.id) + '?mode=gold' }}">Update</a></td>
+                        <td style="width: 100%">
+                            <!-- Allow project owners and admins to update Gold tasks -->
+                            {% if current_user.id in project.owners_ids or current_user.admin %}
```

Comment:
I think the correct behavior is to show the buttons for subadmin owners - regular owners only have read access. In that case https://github.com/bloomberg/pybossa/blob/master/pybossa/api/__init__.py#L443 should also be changed

Reply:
I was basing the change on the [comment](https://jira4.prod.bloomberg.com/browse/RDISCROWD-1635?focusedCommentId=359983&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-359983) in the JIRA ticket, mentioning that:

*"You have to be an admin or project owner to convert a task to gold."*

If this is not correct, we can update the logic accordingly. :)

Reply:
Fixed via https://github.com/bloomberg/pybossa-default-theme/pull/122/commits/1e5af2b362d1351cb5eba653defe0ee558911508